### PR TITLE
Translation of network summary

### PR DESCRIPTION
**Pull request by Peiffap**
Traduction et relecture de la synthèse de network. Je n'ai pas encore eu le cours du coup il y a sûrement des erreurs qui me sont échappées...

### CODE REVIEW

**Comment by blegat** [comment]
cc @Nhoutain @fthuin

**Comment by MartinBraquet** [comment]
28 jours après, personne n'a vérifié cette PR, on pourrait peut-être la valider quand même?

**Comment by Jimvy** [comment]
Je vais la review.
(C'est une mauvaise habitude de merger sans review avant :p )

**Comment by MartinBraquet** [comment]
Parfait, je t'avoue que relire une synthèse de 3000 lignes (d'un cours que je n'ai pas eu) n'est pas trop mon plaisir de fin de soirée ensoleillée... :D

File: src/q5/network-INGI1341/summary/network-INGI1341-summary.tex [hunk 1]
```diff
@@ -472,91 +559,104 @@ \subsection{The network layer}
...
 \end{figure}
 
-\subparagraph{Three-way handshake:} est utilisé contre l'approche naive de la connexion ``aller-retour''.  Celle ci est définie par l'envoi d'un \textbf{CR}, répondu par \textbf{CA} qui répond \textbf{CA}.
-
-\begin{figure}[h]
+\subparagraph{Three-way handshake:} used against the naive approach that is back-and-forth connection. It requires three separate steps:
+\begin{itemize}
+     \item The first host sends the second host a SYN message with its own sequence number $x$, which the second host receives.
+     \item The second host replies with a SYN-ACK message with its own sequence number $y$ and acknowledgment number $x+1$, which the first host receives.
+     \item The first host replies with an ACK message with acknowledgment number $y+1$, which the second host receives and doesn't reply to.
```
Comment (Jimvy) [inline src/q5/network-INGI1341/summary/network-INGI1341-summary.tex]: Not sure if it is mentioned afterwards: the seqnum of this ACK (which may actually contain the first data) is $x+1$, dans the seqnum of the next segment from the second host is $y+1$.
If it seems strange, it is: actually, seqnum don't express segment numbers but rather expected next byte number, and as the handshake has transmitted zero data, we still have the possibility to use it.

File: src/q5/network-INGI1341/summary/network-INGI1341-summary.tex [hunk 2]
```diff
@@ -4,77 +4,105 @@
...
+    data unit that can be communicated in a single network layer transaction.
+    \item[Path MTU]: Maximum transmission unit size on the network between two
+    Internet Protocol hosts.
+    \item[PDU]: \textit{Protocol data unit}. Information that is transmitted
+    as a single unit among peer entities of a computer network.
+    \item[RIFS]: \textit{Reduced Interframe Space}.
+    \item[SDU]: \textit{Service Data Unit}.
+    \item[SIFS]: \textit{Short Interframe Spacing}. Amount of time in
+    microseconds required for a wireless interface to process a received frame
+    and to respond with a response frame.
```
[the remark this answers is not in the document]
Comment (Peiffap) [reply src/q5/network-INGI1341/summary/network-INGI1341-summary.tex]: I'm afraid my understanding of the concepts is not quite profound enough to get where this reference should point to...

File: src/q5/network-INGI1341/summary/network-INGI1341-summary.tex [hunk 3]
```diff
@@ -4,77 +4,105 @@
...
-    \item[SIFS] : \textit{Short Inter Frame Spacing}. Temps pour une interface sans-fil entre la gestion d'une trame reçue et l'envoi de l'acquittement correspondant (temps pour passer de download à upload pour un routeur).  -> CSMA/CA
-    \item[TLV] : \textit{Type Length Value} est un format d'encodage tel que le premier byte est le type, le second la taille totale et puis la valeur.
+    \item[ADSL]: \textit{Asymmetric Digital Subscriber Line}. the working
+    principle here is to have high download speeds but low upload speeds on the
+    consumer's end
+    \item[ARP]: \textit{Address Resolution Protocol}.
+    \item[BPDU]: \textit{Bridge Protocol Data Unit}.
+    \item[CRC]: \textit{Cyclical Redundancy Check}.
+    \item[DIFS]: \textit{DCF Interframe Space}. If a station detects the
+    medium has been continuously idle for DIFS duration, it is then permitted
```
[the remark this answers is not in the document]
Comment (Peiffap) [reply src/q5/network-INGI1341/summary/network-INGI1341-summary.tex]: I agree.

File: src/q5/network-INGI1341/summary/network-INGI1341-summary.tex [hunk 4]
```diff
@@ -4,77 +4,105 @@
...
+    to transmit a frame. DIFS duration can be calculated by the following
+    method: DIFS = SIFS + (2 * Slot time).
+    \item[DNS]: \textit{Domain Name System}. Naming system for associating
+    various information (such as an IP address) with a domain name.
+    \item[EIFS]: \textit{Extended Interframe Space}. Similar to DIFS but is
+    only activated if the last frame contains an error. EIFS = Transmission
+    time of Ack frame at lowest phy mandatory rate + SIFS + DIFS.
```
[the remark this answers is not in the document]
Comment (Peiffap) [reply src/q5/network-INGI1341/summary/network-INGI1341-summary.tex]: Indeed.

File: src/q5/network-INGI1341/summary/network-INGI1341-summary.tex [hunk 5]
```diff
@@ -4,77 +4,105 @@
...
+    \item[HTTP]: \textit{HyperText Transfer Protocol}.
+    \item[IEEE]: \textit{Institute of Electrical and Electronic Engineers}.
```
[the remark this answers is not in the document]
Comment (Peiffap) [reply src/q5/network-INGI1341/summary/network-INGI1341-summary.tex]: You're right, my bad.

File: src/q5/network-INGI1341/summary/network-INGI1341-summary.tex [hunk 6]
```diff
@@ -4,77 +4,105 @@
...
     \cline{1-3}
     CNP3 & TCP/IP & OSI & & & \\
     \hline
-    \multirow{3}{*}{Application} & \multirow{3}{*}{Application}             & Application  & BGP, DHCP, DNS, FTP, HTTP, NFS, NTP, RIP, SMTP, SNMP, Telnet & ADU & \\
+    \multirow{3}{*}{Application} & \multirow{3}{*}{Application}             &
+    Application  & BGP, DHCP, DNS, FTP, HTTP, NFS, NTP, RIP, SMTP, SNMP, Telnet
+    & ADU & \\
     \cline{3-6}
-                                 &                                          & Presentation & MIME, XDR, SSL & & \\
+                                 &                                          &
```
[the remark this answers is not in the document]
Comment (Peiffap) [reply src/q5/network-INGI1341/summary/network-INGI1341-summary.tex]: Blame my OCD...

File: src/q5/network-INGI1341/summary/network-INGI1341-summary.tex [hunk 7]
```diff
@@ -119,350 +151,405 @@ \subsection{Physical Layer}
...
 
 \subsubsection{Characteristic}
 
 It is \textit{not perfect} (unreliable),
-and for us it is like a black box with these characteristics :
+and to us it is like a black box with these characteristics:
 \begin{itemize}
-    \item \textcolor{red}{change} the value of a bit (\textit{car interference electromagnétique});
-    \item deliver \textcolor{red}{more or fewer} bits than requested (\textit{car frequence horloge imprécise})
+    \item \textcolor{red}{changes} the value of a bit (\textit{becasue of
```
[the remark this answers is not in the document]
Comment (Peiffap) [reply src/q5/network-INGI1341/summary/network-INGI1341-summary.tex]: Fixed.

File: src/q5/network-INGI1341/summary/network-INGI1341-summary.tex [hunk 8]
```diff
@@ -119,350 +151,405 @@ \subsection{Physical Layer}
...
-A frame is separated in 3 parts :
+The PDU of Datalink Layer service is a \textbf{frame} (\textit{sequence of bits
+with a particular syntax or structure}) because we want to share data blocks.
+A frame can be separated into 3 parts:
 \begin{itemize}
-    \item \textbf{Header} It contains a flag to tell whether it is an \textcolor{red}{ACK or DATA}, a \textcolor{red}{sequence number} and sometimes the length of the payload.
-  \item \textbf{Payload} It contains the information that needs to be transmitted.
-  \item \textbf{Error Detection Code} It allows the receiver to detect transmission errors.
+    \item \textbf{Header} It contains a flag that tells whether it's an
+    \textcolor{red}{ACK or DATA}, a \textcolor{red}{sequence number} and
```
[the remark this answers is not in the document]
Comment (Peiffap) [reply src/q5/network-INGI1341/summary/network-INGI1341-summary.tex]: True.

File: src/q5/network-INGI1341/summary/network-INGI1341-summary.tex [hunk 9]
```diff
@@ -119,350 +151,405 @@ \subsection{Physical Layer}
...
+When DATA is sent in both directions, an ACK frame and a DATA frame sent by one
+side are sometimes merged into one because an ACK frame does not need a lot of
+bits to do its job. (\textit{Used to reduce the overhead caused by
+acknowledgments}.)
 
 \subsection{Building a network}
-The network layer is used to send packets between two host that cannot directly be relied
-by a cable. Host and router \textbf{send packets}.
+The network layer is used to send packets between two hosts that cannot
+directly be relied by a cable. Host and router \textbf{send packets}.
```
[the remark this answers is not in the document]
Comment (Peiffap) [reply src/q5/network-INGI1341/summary/network-INGI1341-summary.tex]: Oops...

File: src/q5/network-INGI1341/summary/network-INGI1341-summary.tex (hunk 2 above)
[the remark this answers is not in the document]
Comment (Jimvy) [reply src/q5/network-INGI1341/summary/network-INGI1341-summary.tex]: This is mostly a general comment for a global improving of the summary, not a problem with the translation. But the first two § are still valid and don't require any understanding I think ;)

File: src/q5/network-INGI1341/summary/network-INGI1341-summary.tex (hunk 1 above)
Comment (Peiffap) [reply src/q5/network-INGI1341/summary/network-INGI1341-summary.tex]: Bruh.

**Comment by Peiffap** [comment]
Alright, all caught up until here 😊. Thanks for being so thorough!

**Comment by blegat** [comment]
@Jimvy Approved ?

**Comment by Jimvy** [comment]
J'ai pas encore fini de lire depuis la dernière fois ;)

File: src/q5/network-INGI1341/summary/network-INGI1341-summary.tex [hunk 10]
```diff
@@ -684,1820 +855,2050 @@ \subsubsection{Organisation to sharing bandwith}
...
 \paragraph{Selective repeat}
-Avec une option de TCP, on peut activer les \textbf{SACK} qui permettent d'indiquer
-les segments qui ont été reçu hors séquence. On utilise ici souvent Fast retransmit avec.
+One of the \textsc{tcp} options allows activating \textbf{SACKs} which allow us
+to indicate which segment were received out-of-sequence. In this case, Fast
+Retransmit is often used.
 
 \subsubsection{TCP connection release}
-Abrut (\textsc{rst}) ou gracefully (FIN, FIN+ACK)
+Abru[t (\textsc{rst}) or gracefully (FIN, FIN+ACK).
```
[the remark this answers is not in the document]
Comment (Peiffap) [reply src/q5/network-INGI1341/summary/network-INGI1341-summary.tex]: Indeed, I don't... I'm quite new to touch typing (which explains *why* there is a typo) and I switched to US QWERTY after getting really confused with my League of Legends spell names. As a sidenote, the keyboard is in fact your everyday regular AZERTY, but the layout is changed.

File: src/q5/network-INGI1341/summary/network-INGI1341-summary.tex [hunk 11]
```diff
@@ -684,1820 +855,2050 @@ \subsubsection{Organisation to sharing bandwith}
...
+        SCTP & Reliable or & Option infinie & Message mode as & Four-way
+        handshake & chunk \\
+             & partially   & with chunk     & a stream mode
+             &                    & \\
+             &             &                & Multihomed host
+             &                    & \\
         \hline
     \end{tabular}
-    \caption{Difference between UDP, TCP and SCTP}
+    \caption{Difference between UDP, TCP and SCTP.}
```
[the remark this answers is not in the document]
Comment (Peiffap) [reply src/q5/network-INGI1341/summary/network-INGI1341-summary.tex]: If there is more than one, yes.

**Comment by Peiffap** [comment]
@Jimvy How come I can't see which lines you reviewed for the last couple ones? It shows this...

screen shot 2018-07-29 at 23 37 42

**Comment by Jimvy** [comment]
This is pretty strange indeed XD I can't see them either.
I think this is because we've exceeded the maximum number of lines we can safely review before breaking Github.
I will add the line numbers on the blocks.

File: src/q5/network-INGI1341/summary/network-INGI1341-summary.tex [hunk 12]
```diff
@@ -684,1820 +855,2050 @@ \subsubsection{Organisation to sharing bandwith}
...
+The transport layer is where control happens.
 
-\textsc{tcp} control la congestion en agissant sur la window size puisque une connection
-ne peut pas envoyer des données plus vite que  $$\frac{window}{rtt}\quad window=min(swin,rwin)$$
+\textsc{tcp} controls congestion by acting on the window size since a
+connection can't send data faster than \[\frac{\textnormal{window}}{rtt}\quad
+\textnormal{window} = \min(swin,rwin).\]
 
 \paragraph{Congestion window}
-\tex
```
[the remark this answers is not in the document]
Comment (Jimvy) [reply src/q5/network-INGI1341/summary/network-INGI1341-summary.tex]: Line 1723

[the remark this answers is not in the document]
Comment (Jimvy) [reply src/q5/network-INGI1341/summary/network-INGI1341-summary.tex]: 1729

[the remark this answers is not in the document]
Comment (Jimvy) [reply src/q5/network-INGI1341/summary/network-INGI1341-summary.tex]: 1743

[the remark this answers is not in the document]
Comment (Jimvy) [reply src/q5/network-INGI1341/summary/network-INGI1341-summary.tex]: 1767

[the remark this answers is not in the document]
Comment (Jimvy) [reply src/q5/network-INGI1341/summary/network-INGI1341-summary.tex]: 1786

[the remark this answers is not in the document]
Comment (Jimvy) [reply src/q5/network-INGI1341/summary/network-INGI1341-summary.tex]: 1952-1954

[the remark this answers is not in the document]
Comment (Jimvy) [reply src/q5/network-INGI1341/summary/network-INGI1341-summary.tex]: 2073-2074

[the remark this answers is not in the document]
Comment (Jimvy) [reply src/q5/network-INGI1341/summary/network-INGI1341-summary.tex]: 2350
(To be honest, this is not your error, but let's correct it by the way.)

File: src/q5/network-INGI1341/summary/network-INGI1341-summary.tex [hunk 13]
```diff
@@ -684,1820 +855,2050 @@ \subsubsection{Organisation to sharing bandwith}
...
-    \caption{Difference between UDP, TCP and SCTP}
+    \caption{Difference between UDP, TCP and SCTP.}
 \end{table}
 \subsection{Congestion control}
-Le control est effectué dans la transport layer.
+The transport layer is where control happens.
 
-\textsc{tcp} control la congestion en agissant sur la window size puisque une connection
-ne peut pas envoyer des données plus vite que  $$\frac{window}{rtt}\quad window=min(swin,rwin)$$
+\textsc{tcp} controls congestion by acting on the window size since a
```
[the remark this answers is not in the document]
Comment (Jimvy) [reply src/q5/network-INGI1341/summary/network-INGI1341-summary.tex]: 1714 (it still works)

**Comment by Jimvy** [comment]
@Peiffap I've posted a message on the Gitter room, have a look.

File: src/q5/network-INGI1341/summary/network-INGI1341-summary.tex [hunk 14]
```diff
@@ -119,350 +150,429 @@ \subsection{Physical Layer}
...
 
-\paragraph{Note: }
+\paragraph{Note:}
 Since the \textbf{Physical Layer does not reorder} the bits,
 the frames will not be reordered either.
-Providing a reliable service it therefore easier than for the Transport Layer that has to cope with the reordering of packets in the Network Layer (it has to discard packets to old packets and have maximal throughput because of that).
+Providing a reliable service is therefore easier than for the Transport Layer,
+which has to cope with the reordering of packets in the Network Layer
+(it has to discard packets to old packets
+and have maximal throughput because of that).
```
[the remark this answers is not in the document]
Comment (blegat) [reply src/q5/network-INGI1341/summary/network-INGI1341-summary.tex]: No idea :D we can remove if nobody knows what it means :-P

[the remark this answers is not in the document]
Comment (Jimvy) [reply src/q5/network-INGI1341/summary/network-INGI1341-summary.tex]: From what I know, I think it meant "The transport layer has to drop new packets in favor of old packets with the same content, and this consume some throughput." But it is rather sluggish and maybe not worth mentioning here.

File: src/q5/network-INGI1341/summary/network-INGI1341-summary.tex [hunk 15]
```diff
@@ -472,91 +582,135 @@ \subsection{The network layer}
...
+     with acknowledgment number $y+1$,
+     which the second host receives and doesn't reply to.
+\end{itemize}
+The seqnum of that last ACK (which may actually contain the first data) is $x+1$,
+and the seqnum of the next segment from the second host is $y+1$.
+If this seems strange, it is:
+actually, seqnum's don't express segment numbers
+but rather expected next byte numbers.
+As the handshake has transmitted zero data,
+we still have the possibility to use it.
```
[the remark this answers is not in the document]
Comment (Peiffap) [reply src/q5/network-INGI1341/summary/network-INGI1341-summary.tex]: This is a mess 🤦‍♂️

File: src/q5/network-INGI1341/summary/network-INGI1341-summary.tex [hunk 16]
```diff
@@ -472,91 +582,135 @@ \subsection{The network layer}
...
+     \item The first host sends the second host a SYN message
+     with its own sequence number $x$,
+     which the second host receives.
+     \item The second host replies with a SYN-ACK message
+     with its own sequence number $y$
+     and acknowledgment number $x+1$,
+     which the first host receives.
+     \item The first host replies with an ACK message
+     with acknowledgment number $y+1$,
+     which the second host receives and doesn't reply to.
```
[the remark this answers is not in the document]
Comment (Peiffap) [reply src/q5/network-INGI1341/summary/network-INGI1341-summary.tex]: Sorry for not understanding it right away 😄

File: src/q5/network-INGI1341/summary/network-INGI1341-summary.tex [hunk 17]
```diff
@@ -566,115 +720,132 @@ \subsubsection{Connection-oriented}
...
+    needs to be specified to limit SDUs in the bytestream. (The provider
+    assures that bytes arrive in the correct order.)
 \end{itemize}
 
-$\rightarrow$  Le \textbf{Stream-mode}  est utilisé  pour les  reliable
-transport protocols,  et le  numéro de séquence  placé dans  la frame
-correspond à la position du premier \textbf{byte} du payload dans le bytestream.
+$\implies$  The \textbf{Stream-mode} is used for reliable transport protocols,
+and the sequence number placed in the frame corresponds to the position of the
+last payload byte in the bytestream.
```
[the remark this answers is not in the document]
Comment (Peiffap) [reply src/q5/network-INGI1341/summary/network-INGI1341-summary.tex]: Oops...

File: src/q5/network-INGI1341/summary/network-INGI1341-summary.tex [hunk 18]
```diff
@@ -684,1820 +855,2050 @@ \subsubsection{Organisation to sharing bandwith}
...
 \subsubsection{Congestion control}
-\label{congestion}
+\label{sec:congestion}
 
-Pour supprimer la \textbf{congestion collapse}, les hosts doivent réguler leur taux
-de transmission. (\textit{Note qu'il y a d'autre méchanisme pour réguler tel que celui
-basé sur les crédits})
+To remove \strong{congestion collapse}, the hosts have to regulate their
+transmission rate. (\emph{There are other mechanisms to regulate this such as
+the one based around credits.})
```
[the remark this answers is not in the document]
Comment (Peiffap) [reply src/q5/network-INGI1341/summary/network-INGI1341-summary.tex]: I don't think it's in the summary, but the original author seemed to know of one so I assume they didn't pull it out of thin air... Perhaps it appears in the lecture notes? I wouldn't know as I haven't had the class yet.

File: src/q5/network-INGI1341/summary/network-INGI1341-summary.tex [hunk 19]
```diff
@@ -684,1820 +855,2050 @@ \subsubsection{Organisation to sharing bandwith}
...
+    \item \textbf{TS} echo: last received \textsc{ts}.
 \end{itemize}
 
-Comme y'a plus d'ambiguité, on peut mettre à jour la valeur du time ( rtt = 3 sec à l'initialisation).
-\begin{eqnarray*}
-srtt \textrm{ (smoothed rtt mcomputed) } &=&  (\alpha \times srtt) + (1-\alpha) \times rtt) \\
-rto \textrm{ (retransmission timedout) } &=& min(60, max(1, \beta \times srtt))
-\end{eqnarray*}
+Since there is no more ambiguity, the time value can be updated
+($\textnormal{rtt} = \SI{3}{\second}$ at initialisation).
```
[the remark this answers is not in the document]
Comment (Peiffap) [reply src/q5/network-INGI1341/summary/network-INGI1341-summary.tex]: As the American standard prevails in computer science, I'll change it, but this comes down to preference usually.

**Comment by Jimvy** [comment]
I'll have a look back at home.
(Pourquoi je parle anglais ?)

**Comment by Peiffap** [comment]
Alright, sounds good. No rush on this one.
(Maintenant que tu le dis...)

**Comment by blegat** [comment]
Et les étudiants erasmus qui lisent ces messages alors ? :o

**Comment by Jimvy** [comment]
Bon, je viens de me rendre compte qu'en fait le behavior que j'ai probablement décrit à un moment est une option de TCP (Fast Open) assez récente, et que dans le TCP traditionnel on ne peut pas mettre de data dans un paquet contenant le flag SYN.
Donc il y a probablement quelques inexactitudes dans les corrections que j'avais proposées. ^^

File: src/q5/network-INGI1341/summary/network-INGI1341-summary.tex [hunk 20]
```diff
@@ -684,1820 +865,2049 @@ \subsubsection{Organisation to sharing bandwith}
...
 \end{table}
 \subsection{Congestion control}
-Le control est effectué dans la transport layer.
+The transport layer is where control happens.
 
-\textsc{tcp} control la congestion en agissant sur la window size puisque une connection
-ne peut pas envoyer des données plus vite que  $$\frac{window}{rtt}\quad window=min(swin,rwin)$$
+TCP controls congestion by acting on the window size since a
+connection can't send data faster than \[\frac{\textnormal{window}}{rtt}\quad
+\t
```
Comment (Jimvy) [inline src/q5/network-INGI1341/summary/network-INGI1341-summary.tex]: Line 2566:
Could be good to say that in this summary we use interchangeably the terms "switch" and "bridge", although the latter term is more general.

Comment (Jimvy) [inline src/q5/network-INGI1341/summary/network-INGI1341-summary.tex]: Line 2556:
(This is purely cosmetic.)
```suggestion
        $\Rightarrow$ to preserve CSMA/CD: slot time = \SI{5.12}{\micro\second}.
```

Comment (Jimvy) [inline src/q5/network-INGI1341/summary/network-INGI1341-summary.tex]: Line 2518-2530:
This table is ugly as hell.

Comment (Jimvy) [inline src/q5/network-INGI1341/summary/network-INGI1341-summary.tex]: Line 2384:
The goal of LaTeX is to abstract ourselves of this kind of setting and get a consistent document...
```suggestion
    \item \textsc{ESC}: in front of an END if it is part of the information
```

Comment (Jimvy) [inline src/q5/network-INGI1341/summary/network-INGI1341-summary.tex]: Line 2699:
```suggestion
    maximum age is reached, a failure has occurred and the STP has to be
```

Comment (Jimvy) [inline src/q5/network-INGI1341/summary/network-INGI1341-summary.tex]: Line 2560:
```suggestion
Another solution to get better performances in Ethernet LAN is by making hubs
```

Comment (Jimvy) [inline src/q5/network-INGI1341/summary/network-INGI1341-summary.tex]: Line 2542:
```suggestion
    \caption{A hierarchical Ethernet network made of hubs.}
```

Comment (Jimvy) [inline src/q5/network-INGI1341/summary/network-INGI1341-summary.tex]: Line 2433:
```suggestion
to the Internet is done via the Extensible Authentication Protocol (EAP)
```

Comment (Jimvy) [inline src/q5/network-INGI1341/summary/network-INGI1341-summary.tex]: Line 2436:
```suggestion
developped specifications for PPP frame transport on other networks than
```

Comment (Jimvy) [inline src/q5/network-INGI1341/summary/network-INGI1341-summary.tex]: Line 2423 (and other previous lines with the same error):
```suggestion
negotiate a maximum packet size.
```

Comment (Jimvy) [inline src/q5/network-INGI1341/summary/network-INGI1341-summary.tex]: Line 2398:
```suggestion
which \textbf{TCP/IP header compression} techniques were essential.
```

Comment (Jimvy) [inline src/q5/network-INGI1341/summary/network-INGI1341-summary.tex]: Line 2401:
```suggestion
    $\to$ Consecutive segment redundancy is exploited to compress.
```

Comment (Jimvy) [inline src/q5/network-INGI1341/summary/network-INGI1341-summary.tex]: Line 2412 (and other following lines with the same error):
```suggestion
    \item the \textit{Link Control Protocol}: used to negotiate options and to
```

Comment (Jimvy) [inline src/q5/network-INGI1341/summary/network-INGI1341-summary.tex]: Line 2464:
```suggestion
        The beginning of an Ethernet frame begins with a preamble used by the
```

Comment (Jimvy) [inline src/q5/network-INGI1341/summary/network-INGI1341-summary.tex]: Line 2465:
```suggestion
        physical layer to synchronize clocks.
```

File: src/q5/network-INGI1341/summary/network-INGI1341-summary.tex (hunk 18 above)
[the remark this answers is not in the document]
Comment (Jimvy) [reply src/q5/network-INGI1341/summary/network-INGI1341-summary.tex]: I still don't think we've seen such a mechanism... But anyway

File: src/q5/network-INGI1341/summary/network-INGI1341-summary.tex (hunk 1 above)
Comment (Jimvy) [reply src/q5/network-INGI1341/summary/network-INGI1341-summary.tex]: Correction still missing ;)

File: src/q5/network-INGI1341/summary/network-INGI1341-summary.tex (hunk 20 above)
[the remark this answers is not in the document]
Comment (Jimvy) [reply src/q5/network-INGI1341/summary/network-INGI1341-summary.tex]: WTF did Github do here ???

Comment (Peiffap) [reply src/q5/network-INGI1341/summary/network-INGI1341-summary.tex]: I'm afraid it's gonna have to stay that way, I don't really see how to make it look better, and there's bigger issues with the document which I really can't be asked to fix...

**Comment by MartinBraquet** [comment]
Pourquoi c'est en anglais ? C'est pas un cours de bac ?

**Comment by Peiffap** [comment]
Oui, mais le livre de référence écrit par obo est en anglais, et les auteurs avaient mis comme TODO dans la synthèse qu'il fallait la traduire en anglais... Don't shoot the messenger :(

**Comment by Jimvy** [comment]
Même si, pour être honnête, j'aurais préféré une synthèse écrite en français. Même en master ce n'est toujours pas la tasse de thé de lire des trucs en anglais.

**Comment by MartinBraquet** [comment]
Tu la lis encore @Jimvy ?

**Comment by Jimvy** [comment]
Euh oui, re-ping moi si ce n'est pas encore fait à la fin de la semaine

**Pull request merged**